Prox/Prox/Models/Place.swift

```diff
@@ -510,3 +519,26 @@ struct OpenHours {
         return OpenHours.calendar.dateComponents(OpenHours.dateComponentsSet, from: date)
     }
 }
+
+// This could be replaced with a thread-safe dictionary, but I didn't see one in the libs. :(
```

Comment:
You could use NSCache which is thread safe, but that might not allow a tuple as a value